modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -214,6 +225,22 @@ export default class TileLayer extends CompositeLayer {
       return tile.layers;
     });
   }
+
+  getPickObjects() {
```

Comment:
Nit: Add underscore before private method names

Call it `pickObjects`?

Reply:
Done